### DISCUSSION TITLE
Dialog file didnt get all field

### DISCUSSION THREAD

**Discussion by samaremad** [serenity-is/Serenity#5464] [Help] [answered]
hi everyone

i had to add 3 field in users table in database and i updated userRow and userColumn and userForm 
and now i need to add validation to that field in form (in userDialog) but when i write 

this.form.unionID2 

i get error that it must decleared first
can u please help me with that

**Comment by minhhungit** [comment, marked as answer]
are you using .net framework version
did you transform t4 template ?

**Reply by samaremad** [reply]
thank you so much it is worked after the transform
